github/compliance/issue.py

```diff
@@ -85,23 +90,13 @@ def repository_labels(
 
 @github.retry.retry_and_throttle
 def enumerate_issues(
-    component: cm.Component | None,
-    artifact: cm.Artifact | None,
+    labels: frozenset[str],
     known_issues: typing.Sequence[github3.issues.issue.ShortIssue],
-    issue_type: str | None,
```

Comment:
why remove the issue_type?

Reply:
`issue_type` is only used to get repository labels
> yield f'cicd/{issue_type}'

as we pass in labels already, `issue_type` is not required in this func anymore